.github/reusable-build/action.yml

```diff
@@ -0,0 +1,42 @@
+name: Resusable steps to build data-validation
+
+inputs:
+  python-version:
+    description: 'Python version'
+    required: true
+  upload-artifact:
+    description: 'Should upload build artifact or not'
+    default: false
+
+runs:
+  using: 'composite'
+  steps:
+  - name: Set up Python ${{ inputs.python-version }}
+    uses: actions/setup-python@v5
+    with:
+      python-version: ${{ inputs.python-version }}
+
+  - name: Upgrade pip
+    shell: bash
+    run: |
+      python -m pip install --upgrade pip pytest
```

Comment:
Do we need this? This is a reusable workflow, so the particular dependencies should be handled by whatever workflow that calls it.

Reply:
No we don't, good catch.